src/main/java/com/tickets/controller/MovieSessionController.java

```diff
@@ -0,0 +1,61 @@
+package com.tickets.controller;
+
+import com.tickets.model.MovieSession;
+import com.tickets.model.dto.MovieSessionRequestDto;
+import com.tickets.model.dto.MovieSessionResponseDto;
+import com.tickets.service.dto.MovieSessionMapper;
+import com.tickets.service.model.MovieSessionService;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie-sessions")
+public class MovieSessionController {
+    private final MovieSessionService movieSessionService;
+    private final MovieSessionMapper movieSessionMapper;
+
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionMapper movieSessionMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionMapper = movieSessionMapper;
+    }
+
+    @PostMapping
+    public void addMovieSession(@RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+        MovieSession movieSession = movieSessionMapper.dtoToMap(movieSessionRequestDto);
+        movieSessionService.add(movieSession);
+    }
+
+    @GetMapping("/available")
+    public List<MovieSessionResponseDto> getAllAvailable(
+            @RequestParam Long id, @RequestParam @DateTimeFormat(pattern = "dd.MM.yyyy")
```

Comment:
Is this `id` an identifier of MovieSession? If yes why you are using RequestParam? If no, let's rename